### PR TITLE
Update mailing list info

### PR DESCRIPTION
**Pull request by bcipolli**
We got an email to nipy-devel this morning. I thought I'd update the mailing list metadata to what I believe is the current recommendations.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -37,9 +37,17 @@ Current information can always be found at the NIPY website::
 Mailing Lists
 =============
 
-Please see the developer's list::
+For questions on how to use nipy, please see NeuroStars:
```
Comment (matthew-brett) [inline README.rst]: I don't actually use neurostars.  I'm not sure about Alexis or Bertrand, but I can't see them mentioned on neurostars at a quick search.  I personally don't like the idea of splitting user and developer questions.  So, I think we should continue to recommend the (new) mailing list for user questions.

Comment (bcipolli) [inline README.rst]: Sorry, my mistake--I thought this had been agreed on. Here's not the place for such a discussion, I just wanted to update things ASAP.

@matthew-brett how about using github issues? or should everything go through the mailing list?

Comment (matthew-brett) [inline README.rst]: Sure - github issues if it may be a bug, mailing list for general questions or advice about raising an issue.

**Comment by matthew-brett** [comment]
Ben - would you mind finishing up this one?  I think we're nearly ready to go for a release.

**Comment by bcipolli** [comment]
@matthew-brett how's this latest change work for you?

**Comment by matthew-brett** [comment]
How about

```
please see the ``neuroimaging`` mailing list
```

**Comment by bcipolli** [comment]
good call. Fixed.

File: README.rst [hunk 2]
```diff
@@ -37,9 +37,13 @@ Current information can always be found at the NIPY website::
 Mailing Lists
 =============
 
-Please see the developer's list::
+For questions on how to use nipy or on making code contributions, please see the ``neuroimaging`` mailing list:
 
-    http://projects.scipy.org/mailman/listinfo/nipy-devel
+    https://mail.python.org/mailman/listinfo/neuroimaging
+
+For new bug reports, a list of current bugs and planned changes, or to make code contributions, please see the Github issues:
```
Comment (matthew-brett) [inline README.rst]: Oops - sorry - way of rephrasing here so that people don't think they have to go to github issues in order to make code contributions?

Comment (effigies) [inline README.rst]: What about:

    Github issues may also be used to report bugs, contribute code, or see a list of known bugs and planned changes:

File: README.rst [hunk 3]
```diff
@@ -37,9 +37,13 @@ Current information can always be found at the NIPY website::
 Mailing Lists
 =============
 
-Please see the developer's list::
+For questions on how to use nipy or on making code contributions, please see the ``neuroimaging`` mailing list:
 
-    http://projects.scipy.org/mailman/listinfo/nipy-devel
+    https://mail.python.org/mailman/listinfo/neuroimaging
+
+Github issues may also be used to report bugs, or to see a list of known bugs and planned changes:
```
Comment (effigies) [inline README.rst]: Sorry, grammar tweak: No comma, since the last part wouldn't be a full sentence on its own, and the list is now two items long.

**Comment by bcipolli** [comment]
@effigies @matthew-brett how's the latest? I even squashed the commits... embarrassing. :)

**Comment by effigies** [comment]
What, you don't want 6 commits for a +6/-2?

LGTM.

File: README.rst [hunk 4]
```diff
@@ -37,9 +37,13 @@ Current information can always be found at the NIPY website::
 Mailing Lists
 =============
 
-Please see the developer's list::
+For questions on how to use nipy or on making code contributions, please see the ``neuroimaging`` mailing list:
 
-    http://projects.scipy.org/mailman/listinfo/nipy-devel
+    https://mail.python.org/mailman/listinfo/neuroimaging
+
+Github issues may also be used to report bugs or to see a list of known bugs and planned changes:
```
Comment (matthew-brett) [inline README.rst]: I know this is getting a bit embarrassing, but how about:

```
Please report bugs at github issues:

    https://github.com/nipy/nipy/issues

You can see the list of current proposed changes at:

    https://github.com/nipy/nipy/pulls
```

**Comment by bcipolli** [comment]
@matthew-brett made the changes you requested.

**Comment by matthew-brett** [comment]
Sorry to be so boring about it - thanks for keeping going.

**Pull request merged**